packages/jest-resolve/src/index.js

```diff
@@ -91,8 +91,8 @@ class Resolver {
 
   static findNodeModule(path: Path, options: FindNodeModuleConfig): ?Path {
     const resolver = options.resolver
-      /* $FlowFixMe */
-      ? require(options.resolver)
+      ? /* $FlowFixMe */
```

Comment:
I'm curious, is this a regression from before?

It indeed looked better before.